src/blockdev.rs

```diff
@@ -1556,6 +1556,7 @@ fn count_partition_by_label(label: &str, devices: &[Device]) -> usize {
 }
 
 pub fn check_single_partition(label: &str) -> Result<bool> {
+    runcmd!("partprobe")?;
```

Comment:
on s390x with 2 paths to same SCSI (`multipath` was `off`) i hit the issue during `coreos-gpt-setup`:
- new UUID was generated for `boot`, so `sda` had it, but `sdb` had some other from preious install

So `partprobe` forces system to update its/kernel info regarding UUID

Reply:
We discussed this out-of-band today.

I think it would make sense to have `coreos-gpt-setup` do the retriggering instead. Rather than adding `partprobe`, it could use `lsblk` + `blockdev --rereadpt` or call out to e.g. `rdcore rereadpt [DEV]` since we already have the code to trigger partition reloads via `ioctl`.